src/main/scala/io/iohk/ethereum/keystore/EncryptedKeyJsonCodec.scala

```diff
@@ -0,0 +1,88 @@
+package io.iohk.ethereum.keystore
+
+import java.util.UUID
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.Address
+import io.iohk.ethereum.keystore.EncryptedKey._
+import org.json4s.JsonAST.{JObject, JString, JValue}
+import org.json4s.JsonDSL._
+import org.json4s.native.JsonMethods._
+import org.json4s.{CustomSerializer, DefaultFormats, Extraction}
+import org.spongycastle.util.encoders.Hex
+
+import scala.util.Try
+
+object EncryptedKeyJsonCodec {
+
+  private val byteStringSerializer = new CustomSerializer[ByteString](_ => (
+    { case JString(s) => ByteString(Hex.decode(s)) },
+    { case bs: ByteString => JString(Hex.toHexString(bs.toArray)) }
+  ))
+
+  private implicit val formats = DefaultFormats + byteStringSerializer
+
+  private def asHex(bs: ByteString): String =
+    Hex.toHexString(bs.toArray)
+
+  def toJson(encKey: EncryptedKey): String = {
+    import encKey._
+    import cryptoSpec._
+
+    val json =
+      ("id" -> id.toString) ~
+      ("address" -> asHex(address.bytes)) ~
+      ("version" -> version) ~
```

Comment:
From the wiki:

> Version is intrinsically numeric yet is a string (structured versioning would be possible with a string, but can be considered out of scope for a rarely changing configuration file format).

I sometimes don't know why I even bother with reading the docs... Yes, it should be numeric.